client/webserver/locales/en-us.go

```diff
@@ -364,8 +362,12 @@ var EnUS = map[string]string{
 	"Backup App Seed":                 "Backup App Seed",
 	"seed_backup_implore": "It's highly recommended that you back up your application seed now. " +
 		"The application seed is critical to recovering your fidelity bonds and native wallets.",
-	"Backup Now":             "Backup Now",
-	"Skip this step for now": "Skip this step for now",
-	"save_seed_instructions": "Save the following seed somewhere safe. Do not share this with anybody.",
-	"Done":                   "Done",
+	"Backup Now":                "Backup Now",
+	"Skip this step for now":    "Skip this step for now",
+	"save_seed_instructions":    "Save the following seed somewhere safe. Do not share this with anybody.",
+	"Done":                      "Done",
+	"Reset App Password":        "Reset App Password",
+	"Forgot App Password":       "Forgot App Password",
```

Comment:
`Forgot App Password` is never used, 

and I also think it's a bit more intuitive than `Reset App Password`